controllers/DataPresentationAdmin.js

```diff
@@ -0,0 +1,74 @@
+import {AdminController} from 'nxus-admin'
+import {templater} from 'nxus-templater'
+import {actions} from 'nxus-web'
+import {router} from 'nxus-router'
+import _ from 'underscore'
+
+import * as field from '../fieldConstants'
+
+export default class DataPresentationAdmin extends AdminController {
+  constructor() {
+    super({
+      modelIdentity: 'datasets-datapresentation',
+      instanceTitleField: 'DataPresentation'
+    })
+    //custom datapresentation edit and create form template
+    templater.replace().template(__dirname+"/../templates/datasets-datapresentation-form.ejs", this.pageTemplate, this.templatePrefix+"-edit")
+    templater.replace().template(__dirname+"/../templates/datasets-datapresentation-form.ejs", this.pageTemplate, this.templatePrefix+"-create")
+  }
+
+
+  edit(req, res, query) {
+    return super.edit(req, res, query)
+    .then( (editContext) => {
+      let instance = editContext.object
+      this.log.debug( "editing datapresentation id:  ", instance.id)
+      return [this.models['datasets-dataset'].find(), editContext]
+    }).spread( (datasets, context) => {
+      this.log.debug( "edit handling datasets  ", _.pluck(datasets, "id"))
+      let retObj = {
+        datasets: datasets,
+        ...context
+      }
+      this.log.debug( "returning datapresentation edit context:  ", retObj)
+      return retObj
+    })
+  }
+
+  save(req, res) {
+    let presentId = req.params.id
+    this.log.debug("save presentation id: ", presentId)
+    this.log.debug(" body: ", req.body.fieldIds)
+    if (presentId) {

```

Comment:
Er, though that assumes you have some key field to search by besides ID - it probably reasonably should accept a non-object criteria object and act as this does (just doing a create without the ID) if it's null.